docs/Getting-Started.md

```diff
@@ -81,11 +84,12 @@ const fastify = require('fastify')()
 
 fastify.register(require('./our-first-route'))
 
-fastify.listen(3000, function (err) {
+fastify.listen(3000, function (err, address) {
   if (err) {
     fastify.log.error(err)
     process.exit(1)
   }
+  console.log(`server listening on ${address}`)
```

Comment:
Use `fastify.log.info` instead of `console.log`.

Reply:
fixed